app/scripts/controllers/reviewRegistration.js

```diff
@@ -121,14 +118,28 @@ angular
       });
     }
 
+    function getRegistrantType(typeId) {
```

Comment:
I think we could .find instead of using lodash here: 
```suggestion 
function getRegistrantType(typeId) {
      return conference.registrantTypes.find(type => type === typeId);
    }
```

Reply:
I've just tried to replace it with .find but it failed on CI, looks like there is no such function there:
> TypeError: undefined is not a function (evaluating 'conference.registrantTypes.find(function (type) {
Should add a polyfill for this function? I reverted it to use lodash for now.

Reply:
Hmm, .filter is actually what I should have put but the lodash should be fine honestly. We use it everywhere else.